public/index.html

```diff
@@ -3,11 +3,47 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie Account</title>
+    <link rel="stylesheet" href="/default/css/default.css">
     <script src="/hoodie/client.js"></script>
   </head>
   <body>
-    <p>
-       Open browser console and play with <code>hoodie.account</code>.
-    </p>
+
+    <div data-hide="false" class="generic-loader-wrap"></div>
+
+    <section data-hide="true" class="input-forms">
+      <h3>Sign In</h3><br>
+      <input name="login-username-field" type="text" placeholder="Username"><br>
```

Comment:
Could we add labels to all form inputs, it's more accessible. Placeholders dissapear on focus.